### DISCUSSION TITLE
It's now 2022 and Week 7 source isn't here

### DISCUSSION THREAD

**Discussion by meltfrommeltland** [FunkinCrew/Funkin#1919] [General]
It's now 2022 and Week 7 source code isn't here. Yeesh.

**Comment by ACrazyTown** [comment]
That's because the Week 8 stuff got added before Week 7 was done.

The new update was announced for Q1 2022, wait a little longer

**Reply by AbnormalPoof** [reply]
FYI, Q1 is January 1st-March 31st.

**Reply by ACrazyTown** [reply]
> FYI, Q1 is January 1st-March 31st.

ye ye i know

**Comment by SilverT4** [comment]
week 7 source will be included with week 8, i believe.

**Comment by ninjamuffin99** [comment]
eventually

**Reply by meltfrommeltland** [reply]
ok

**Reply by trollguy62** [reply]
hi

**Comment by NokioPyro** [comment]
They will probably send out the source on the next update, as i know it will be around april or march.
And The source is still newgrounds only right now.

**Reply by meltfrommeltland** [reply]
...this was posted over a month ago with several answers.
